felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Faktainfo.kt

```diff
@@ -1,15 +1,18 @@
 package no.nav.familie.kontrakter.felles.tilbakekreving
 
+import com.fasterxml.jackson.annotation.JsonFormat
+
 data class Faktainfo(
         val revurderingsårsak: String,
         val revurderingsresultat: String,
         val tilbakekrevingsvalg: Tilbakekrevingsvalg,
         val konsekvensForYtelser: Set<String> = emptySet()
 )
 
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
```

Comment:
Treng me JsonFormat.Shape.OBJECT?
Vil ikkje fagsystema bruke koden?

Reply:
Nei, fagsystemet bruker ikke den. Men jeg gjenbrukte samme DTO for å returnere data til frontend. Derfor brukte jeg det.